library/dnet_common.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2008+ Evgeniy Polyakov <zbr@ioremap.net>
+ * Copyright 2015+ Yandex
```

Comment:
Copyrights are added to people for new file creation or major contribution, neither of it is present in the commit.